resin/auth.py

```diff
@@ -119,6 +148,12 @@ def get_token(self):
         Raises:
             InvalidOption: if not logged in and there is no token in Settings.
 
+        Examples:
+            # If you are logged in.
```

Comment:
Nope, it's not :p One example: https://github.com/nghiant2710/resin-sdk-python/blob/master/DOCUMENTATION.md#examples-10